packages/eventual-send/test/test-eventual-send.js

```diff
@@ -159,8 +159,11 @@ test('new HandledPromise expected errors', async t => {
     get(o, _key) {
       return o;
     },
-    applyMethod(o, key, _args) {
-      return key;
+    applyFunction(o, args) {
+      return args.join(',');
+    },
+    applyMethod(o, key, args) {
+      return [key, ...args].join(',');
```

Comment:
I think I called these `apply` and `invoke` in Q, but I’m not sure they match exactly. `apply` was only applicable to promises for functions and `invoke` was applicable only for promises for objects with methods/functions.

Is the intention here that both of these apply to objects with functions, but `applyMethod` cares about the receiver object?

Reply:
`applyMethod` avoids a round trip between `get`ting the method and `applyFunction`ing it.  It also incidentally permits setting `this` correctly for the receiving method (though defensive programs will not rely on this feature).

These methods will be renamed once more of https://github.com/tc39/proposal-eventual-send is implemented.  They're legacy baggage for now.